types/choose_sequence/index.js

```diff
@@ -0,0 +1,168 @@
+const getUniqueId = require('../../utils/getUniqueId');
+
+function addCodeTags(inputText) {
+  const regexp = /```(.|[^`]+)```/gs;
```

Comment:
let's better use some md2html package (like `remark` we tried in parser)

Reply:
Ok I'll try.

Reply:
and looks like we agreed (in chat) to use html here (just sum up)